test/CMakeLists.txt

```diff
@@ -419,7 +419,9 @@ castxml_test_output(ArrayType-incomplete)
 castxml_test_output(Class)
 castxml_test_output(Class-abstract)
 castxml_test_output(Class-annotate)
+set(castxml_test_output_custom_start --castxml-start start,end)
 castxml_test_output(Class-deprecated)
+unset(castxml_test_output_custom_start)
```

Comment:
I'd prefer not to need this special case for start/end.  Instead please just add another `Class-deprecated-long` case.